src/Components/Classes/List/list.module.css

```diff
@@ -84,3 +84,45 @@
   border-radius: 5px;
   cursor: pointer;
 }
```

Comment:
The styles below this comment are not being used and serve no purpose. They should be deleted to maintain cleaner code.